OneCommunityGlobal/HighestGoodNetworkApp
# Jerry - Unit test AssignBadge.jsx

# Description
Create Unit Tests for AssignBadge.jsx

## Related PRS (if any):
N/A

## Main changes explained:
- Added a new file: `src/components/Badge/__tests__/AssignBadge.test.js`

## How to test:
1. On the `development` branch of HGNRest (backend), run the server by running the following command: `npm i && npm run build && npm start`
1. check into current branch on HighestGoodNetworkApp (frontend).
2. do `npm install`
3. Run `npm test AssignBadge.test.js` to run the unit tests

## Screenshots or videos of changes:

<img width="676" alt="Screenshot 2024-03-09 at 3 59 05 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/55968519/36cbe2d8-431b-4601-863c-6c162006851e">
